### PR TITLE
Update README

### LINKED ISSUE #668: Simplify README and update wiki
The README seems too long to be an effective introduction to the repo, and plenty of information presented in it is out-dated. I believe that in README, only the most important and up-to-date information should be presented.

I'd like to restructure (mostly simplify) the README to make everything brief and put links to wiki as the detailed documentation, i.e. the following points:

* Quick start
* Features
* Questions, bug reports, feature requests, and contribution

The original content should be mostly moved to wiki and I'll put links in "Features". Also, extensions we recommend users to install with vscode-R such as vscode-r-lsp and VSCode-R-Debugger should be introduced in README, so that user could get full R development experience including language service and debugging capabilities immediately.

### PR DESCRIPTION
**Pull request by renkun-ken**
Closes #668

- [x] README
- [x] wiki pages

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,387 +1,58 @@
...
-* A: Please open your folder that has R source file
-
-* Q: About code formatter, completion, definition...
-* A: Please visit to the language server [issues](https://github.com/REditorSupport/languageserver/issues)
-
-Other past questions can be found from [StackOverflow](https://stackoverflow.com/questions/tagged/visual-studio-code+r) or [issues](https://github.com/Ikuyadeu/vscode-R/issues)
+## Features
 
-The R logo is © 2016 The R Foundation
+* Extended syntax highlighting for R, R Markdown and R Documentation.
```
Comment (Ikuyadeu) [inline README.md:35]: Could you make the space like a python extension for each item?
All of them are many great contributions but look a bit complicated.

**Comment by renkun-ken** [comment]
The wiki pages are mostly updated. I guess we could merge this PR now.

I'll still add content to wiki over time.

**Comment by Ikuyadeu** [comment]
LGTM
Thank you @renkun-ken !

**Comment by ManuelHentschel** [comment]
The updated readme and wiki entries look really nice, thanks a lot!

**Pull request merged**